### PR TITLE
Export timestamp-container utilities

### PR DESCRIPTION
**Pull request by OlgaLyubin**
Resolves #NUMBER

**Overall change:** 
Export `timestampUtilities` from `@bbc/timestamp-container` so that `formatUnixTimestamp ` can be used for Radio Schedules in https://github.com/bbc/psammead/pull/3074.

**Code changes:**
- Exported `timestampUtilities` by creating a `package.json`
- Added export to README

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by DenisHdz — APPROVED** [review]
LGTM, just wondering if we should create a `utilities` folder inside the `psammead-timestamp-container` which includes the `timestampUtilities.js` and `timestampUtilities.test.js` to keep the files more organized.

**Comment by OlgaLyubin** [comment]
> LGTM, just wondering if we should create a `utilities` folder inside the `psammead-timestamp-container` which includes the `timestampUtilities.js` and `timestampUtilities.test.js` to keep the files more organized.

Good point, that would be a cleaner structure. Will update the PR

File: packages/containers/psammead-timestamp-container/src/index.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {
   isValidDateTime,
   unixTimestampToMoment,
   formatUnixTimestamp,
-} from './timestampUtilities';
+} from './Utilities/timestampUtilities';
```
Comment (DenisHdz) [inline packages/containers/psammead-timestamp-container/src/index.jsx]: Usually, the utility folders are lowercase, do you mind making this change?

Comment (OlgaLyubin) [reply packages/containers/psammead-timestamp-container/src/index.jsx]: I have updated the imports and renamed the folder, but for some reason GitHub doesn't register the change from `Utilities` to `utilities` and it did not update the name even though locally everything is correct. Currently looking into it.

**Review by EinsteinNjoroge — APPROVED** [review]
LGTM

File: packages/containers/psammead-timestamp-container/src/index.jsx (hunk 1 above)
Comment (OlgaLyubin) [reply packages/containers/psammead-timestamp-container/src/index.jsx]: This is done now :) Git doesn't like case sensitivity

**Review by DenisHdz — APPROVED** [review]
LGTM

**Pull request merged**